backstop/tests/table.html

```diff
@@ -3798,5 +3822,32 @@ <h2>Empty state</h2>
         });
       });
     </script>
+
+    <script>
+      let reorderStartIndex = -1;
+      const onDragStart = (event) => {
+        const column = event.currentTarget;
+        reorderStartIndex = Array.from(column.parentElement.childNodes).indexOf(column);
+      };
+      const onDragOver = (event) => {
+        event.preventDefault();
+        const column = event.currentTarget;
+        const columnIndex = Array.from(column.parentElement.childNodes).indexOf(column);
+        if (columnIndex === reorderStartIndex) {
+          return;
+        }
+        column.classList.add(columnIndex > reorderStartIndex ? 'iui-reorder-column-right' : 'iui-reorder-column-left');
+      };
+      const onDragLeave = (event) => {
```

Comment:
Can't this class and `onDrop` be combined into one function?

Reply:
You mean to have one function for `onDragLeave` and `onDrop`?

Reply:
Reusing.